pubnub/lib/src/core/crypto/crypto.dart

```diff
@@ -12,7 +12,7 @@ class CryptoException extends PubNubException {
 }
 
 /// @nodoc
-abstract class ICryptoModule {
+abstract class ILegacyCryptor {
```

Comment:
Android Studio shows me this class is never used. Is it true? :)

Reply:
Yeah!  We do not need it.  Earlier I was thinking to have a different interface for LegacyCryptor. But then I was able to figure out that  we can unify the signature and have only one interface that's `ICryptor`.

removing it.